app/src/ui/rebase/rebase-flow.tsx

```diff
@@ -223,12 +294,21 @@ export class RebaseFlow extends React.Component<
     // detects and handles the state transition after a period of time to ensure
     // the UI shows _something_ before closing the dialog
     this.setState(prevState => {
+      let currentCommitSummary: string | undefined = undefined
+
+      const { rebasePreview: rebaseStatus } = prevState
+
+      if (rebaseStatus !== null && rebaseStatus.kind === ComputedAction.Clean) {
+        const { commits } = rebaseStatus
+        currentCommitSummary = commits[commits.length - 1].summary
```

Comment:
This seems ripe for a `Cannot read property 'summary' of undefined` error one day. Could we add a check that `commits.length > 0` please?

Reply:
Addressed in 751cc7f44e4524ed0bcd59b1a4501c43cb759e9d